files/sharepointplus/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "sharepointplus",
+  "repo": "aymkdn/sharepointplus",
+  "files": {
+    "basePath": "jsdelivr/"
```

Comment:
No need for the unminified version anymore. And I decided to use a more
correct version number, it's why it's now v3.12